lib/asciidoctor/standoc/inline.rb

```diff
@@ -148,6 +149,28 @@ def inline_quoted(node)
           end
         end.join
       end
+
+      def check_latexml_version
+        recommended_version = "0.8.4"
```

Comment:
Can you extract this to a constant placed somewhere in some config file?

Reply:
@ronaldtse  Will be `Metanorma:: Standoc::RECOMMENDED_LATEXML_VERSION` good place for this?

Reply:
How about this:

1. Make a hash called `Metanorma::Standoc::Requirements`
2. Make an abstract class called `Metanorma::Standoc::Requirement`
3. Make a class called `Metanorma::Standoc::LatexmlRequirement < Metanorma::Standoc::Requirement` that implements this check
4. Place this class into `Metanorma::Standoc::Requirements[:latexml] = Metanorma::Standoc::LatexmlRequirement`

Reply:
@ronaldtse done